src/extensions/cellpack/state.ts

```diff
@@ -95,9 +100,101 @@ const ParseCellPack = PluginStateTransform.BuiltIn({
                     }
                 }
             }
+            const { options } = cell;
+            if (!params.modelFile && options) {
+                if (options.resultfile) {
+                    const asset = await plugin.runTask(plugin.managers.asset.resolve(Asset.getUrlAsset(plugin.managers.asset, `${params.baseUrl}/results/${options.resultfile}`), 'binary', true));
```

Comment:
You want to be able to reuse this asset in multiple transforms from the parent ``Cell``?